components/calculation/PopulationComparisonTable.tsx

```diff
@@ -47,45 +47,195 @@ export default function PopulationComparisonTable({ patientId }: PopulationCompa
     return measureBundle.content?.entry?.find(e => e.resource?.resourceType === 'Measure')?.resource as fhir4.Measure;
   }, [measureBundle]);
 
+  /**
+   * Creates a key that can be used to represent the population associated with a particular population result.
+   * This key can be used to show the user what population the results are relevant for. It generally uses the
+   * criteria expression, but makes adjustments for measure observations where the expression may not be unique.
+   */
+  function keyForResult(result: PopulationResult, group: DetailedPopulationGroupResult | undefined) {
+    let key = result.criteriaExpression as string;
```

Comment:
I would still suggest not using the criteria expression and suggest using the `populationType` mapped to a short hand. ex. `initial-population` -> `IPP`.  It is decently common for measures to use the same define statement for both IPP and DENOM. So in this situation you would see "Initial Population" as the header for two columns.

Reply:
I think OBSERV, OBSERV-NUMER, OBSERV-DENOM. would work well